src/client/entity-editor/validators/base.js

```diff
@@ -20,9 +20,12 @@
 
 import {Iterable} from 'immutable';
 import _ from 'lodash';
+import moment from 'moment';
 import validator from 'validator';
 
 
+const dateFormats = ['YYYY-MM-DD', 'YYYY-MM', 'YYYY'];
```

Comment:
A constant called VALID_DATE_FORMATS might be better here (if the linter allows that name).